doc/WebSite/Zero/Permission-Management.md

```diff
@@ -1,23 +1,23 @@
 #### About Authorization
 
-It's strongly suggested to read [authorization
-documentation](/Pages/Documents/Authorization) before this document.
+We strongly recommend you read the [authorization
+documentation](/Pages/Documents/Authorization) before this one.
 
 ### Introduction
 
-Module-zero implements **IPermissionChecker** interface of ASP.NET
-Boilerplate's authorization system. To define and check permissions, you
-can see [authorization document](/Pages/Documents/Authorization). In
-this document, we will see how to grant permissions for roles and users.
+Module-zero implements the **IPermissionChecker** interface of ASP.NET
```

Comment:
@hikalkan Standardise `Module-Zero` or `Module Zero`?

Reply:
@hikalkan

Reply:
@hikalkan

Reply:
No problem, but if it should be standardised, we can use "Module Zero".

Reply:
@acjh @hikalkan Check it out now and let me know if there's anything else you need.